reference/config_files/credentials.rst

```diff
@@ -0,0 +1,64 @@
+.. _reference_config_files_credentials:
+
+credentials.json
+================
+
+.. include:: ../../common/experimental_warning.inc
+
+
+Conan can authenticate against its Conan remote servers with the following:
+- Interactive command line, when some server launches an unauthorized error, the Conan client will ask for user/password interactively and retry.
+- With the ``conan remote login`` command, authentication can be done with argument passing, or interactively.
+- With the environment variables ``CONAN_LOGIN_USERNAME`` for all remotes (``CONAN_LOGIN_USERNAME_{REMOTE}`` for an individual remote) and ``CONAN_PASSWORD`` (``CONAN_PASSWORD_{REMOTE}`` for an individual remote), Conan will not request interactively in the command line when necessary, but will take the values from the environment variables as if they were provided by the user.
+- With a ``credentials.json`` file put in the Conan cache.
+
+This section describes the usage of ``credentials.json`` file.
+
+
+This file has the following format, in which every ``credentials`` entry should have a ``remote`` name, matching the name defined in ``conan remote list``.
+Then, the ``user`` and ``password`` fields.
+
+.. code-block:: json
+
```

Comment:
I would add a comment somewhere, like a warning saying that is not recommended to store plain text credentials and to better use environment with the jinja syntax

Reply:
Users requesting this file requested it because they said it was more secure to have credentials in a file than having them in environment variables...